code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -209,16 +209,18 @@
 			robotic_limb_damage += limb.get_damage(stamina=FALSE)
 
 	if(mob_occupant.health >= mob_occupant.getMaxHealth() - robotic_limb_damage) // Don't bother with fully healed people. Now takes robotic limbs into account.
-		if(C)
-			if(C.all_wounds)
-				if(!treating_wounds) // if we have wounds and haven't already alerted the doctors we're only dealing with the wounds, let them know
-					treating_wounds = TRUE
-					playsound(src, 'sound/machines/cryo_warning.ogg', volume) // Bug the doctors.
-					var/msg = "Patient vitals fully recovered, continuing automated wound treatment."
-					radio.talk_into(src, msg, radio_channel)
-			else // otherwise if we were only treating wounds and now we don't have any, turn off treating_wounds so we can boot 'em out
-				treating_wounds = FALSE
-
+		var/has_cryo_wound = FALSE
+		if(C && C.all_wounds)
+			for(var/datum/wound/wound in C.all_wounds)
```

Comment:
```suggestion
			for(var/datum/wound/wound as anything in C.all_wounds)
```

Reply:
What would this change do exactly?

Reply:
It removes the check of all things in ``all_wounds`` to ensure they are a ``/datum/wound``, saving some time.